src/scripts/ic8/hiv/hiv_checks.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, *args, **kwargs):
 
     # Load the files
     model_results = ModelResultsHiv(
-        path_to_data_folder / "IC8/modelling_outputs/hiv",
+        path_to_data_folder / "IC8/modelling_outputs/hiv/18Jul2024",
```

Comment:
I would recommend using the data format YYYYMMDD (e.g. 20240718 or 2024_07_18). This way you can sort the folder by name and it will be in data order, earliest to latest.